assets/component-newsletter.css

```
@@ -19,12 +19,10 @@
   width: 100%;
 }
 
-.newsletter-form__field-wrapper .field__input {
-  padding-right: 5rem;
-}
-
-.newsletter-form__field-wrapper .field {
+.newsletter-form__field-wrapper .field-new {
   z-index: 0;
+  display: flex;
+  gap: 1rem;
```

```suggestion
  gap: 1.6rem;
```

The gap `1rem` is calculating at `10px` and the design is about `16px`, so I think this needs to be `1.6rem`. Please check the spacing after changing.